packages/account/src/Components/self-exclusion/__tests__/self-exclusion-article-content.spec.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import SelfExclusionArticleContent from '../self-exclusion-article-content';
+import SelfExclusionContext from '../self-exclusion-context';
+import { Popup } from '@deriv/components';
+
+Popup.Overlay = jest.fn(() => <div>Trading limits</div>);
+let mockContext = {};
```

Comment:
need to update most of the variable definitions to follow snake_case as we do all over Deriv app.